numpy/polynomial/polynomial.py

```diff
@@ -1241,6 +1283,27 @@ def polyvander3d(x, y, z, deg):
 
     .. versionadded:: 1.7.0
 
+    Examples
+    --------
+    >>> from numpy.polynomial import polynomial as P
+    >>> x, y, z, deg  = [-1, 2, 1], [1, -2, -3], [2, 2, 5], [2, 2, 1]
+    >>> V = P.polyvander3d(x=x, y=y, z=z, deg=deg)
+    array(array([[  1.,   2.,   1.,   2.,   1.,   2.,  -1.,  -2.,  -1.,  -2.,  -1.,
+                   -2.,   1.,   2.,   1.,   2.,   1.,   2.],
+                 [  1.,   2.,  -2.,  -4.,   4.,   8.,   2.,   4.,  -4.,  -8.,   8.,
+                   16.,   4.,   8.,  -8., -16.,  16.,  32.],
+                 [  1.,   5.,  -3., -15.,   9.,  45.,   1.,   5.,  -3., -15.,   9.,
+                   45.,   1.,   5.,  -3., -15.,   9.,  45.]])
+    
+    To verify some values one can do the following computations:
+    
+    >>> from numpy.polynomial import polynomial as P
+    >>> x, y, z  = np.array([-1, 2, 1]), np.array([1, -2, -3]), np.array([2, 2, 5])
+    >>> deg = [2, 2, 1]
+    >>> V = P.polyvander3d(x=x, y=y, z=z, deg=deg)
+    >>> i, j, k = 0, 1, 1
+    >>> V[:, (deg[1]+1)*(deg[1]+1)*i + (deg[2]+1)*j + k] == x**i * y**j * z**k
```

Comment:
```suggestion
    >>> V[:, (deg[1]+1)*(deg[2]+1)*i + (deg[2]+1)*j + k] == x**i * y**j * z**k
```
If I'm reading the documentation correctly. Would be easier to compare the two if `deg` were defined in terms of some variables `l`, `m`, and `n`.